sdks/go/pkg/beam/runners/universal/runnerlib/stage.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // Stage stages the worker binary and any additional files to the given
 // artifact staging endpoint. It returns the commit token if successful.
-func Stage(ctx context.Context, id, endpoint, binary string, files ...artifact.KeyedFile) (string, error) {
+func Stage(ctx context.Context, id, endpoint, binary, st string, files ...artifact.KeyedFile) (string, error) {
```

Comment:
Change comment to speak about "retrieval token", and give a name to the "string" result?